### PR TITLE
Remove autocomplete dropdown on the search query field.

### LINKED ISSUE #131: Remove native autocomplete from the search query field.
The autocompletion dropdown appears over the top of the search results, like this:

Screenshot 2020-05-24 at 07 36 12@2x

Layering over the first search result isn't ideal! See more discussion of this on [#107](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/issues/107#issuecomment-633188893).

### PR DESCRIPTION
**Pull request by daveverwer**
Fixes #131 

Ignore the weird branch name, that's just how this branch started 😂

### CODE REVIEW

**Comment by daveverwer** [comment]
There are a couple of other bits of JS reorganisation and clean up in here, but the main thing is to fix the autocomplete dropdown issue.

**Comment by daveverwer** [comment]
Need to fix a small CSS issue with this, changing back to Draft for now.

**Comment by daveverwer** [comment]
Now it's ready!

File: Tests/AppTests/__Snapshots__/WebpageSnapshotTests/test_HomeIndexView.1.txt [hunk 1]
```diff
@@ -36,7 +36,7 @@
       <div class="inner">
         <h3>The place to find Swift packages.</h3>
         <form>
-          <input type="text" id="query" placeholder="Search" spellcheck="false" autofocus="true"/>
+          <textarea id="query" placeholder="Search" spellcheck="false" data-gramm="false" autofocus="true" rows="1"></textarea>
```
Comment (finestructure) [inline Tests/AppTests/__Snapshots__/WebpageSnapshotTests/test_HomeIndexView.1.txt:39]: Out of curiosity: what is `data-gramm`? I can't find any reference about that.

Comment (finestructure) [reply Tests/AppTests/__Snapshots__/WebpageSnapshotTests/test_HomeIndexView.1.txt:39]: Oh, I see it in the commit message now!

**Pull request merged**